public/button.js

```diff
@@ -11,14 +11,24 @@ const retrieveStart = () => {
   );
 };
 
+const secondsToDhms = (seconds=0) => {
+  seconds = Number(seconds);
+  const d = Math.floor(seconds / (3600*24));
+  const h = Math.floor(seconds % (3600*24) / 3600);
+  const m = Math.floor(seconds % 3600 / 60);
+  const s = Math.floor(seconds % 60);
+  
+  return `${d}d ${h}h ${m}m ${s}s`;
+  }
+
```

Comment:
Indentation of closing bracket doesn't seem right

```suggestion
const secondsToDhms = (seconds=0) => {
  seconds = Number(seconds);
  const d = Math.floor(seconds / (3600*24));
  const h = Math.floor(seconds % (3600*24) / 3600);
  const m = Math.floor(seconds % 3600 / 60);
  const s = Math.floor(seconds % 60);
  
  return `${d}d ${h}h ${m}m ${s}s`;
}

```